interfaces/builtin/mir.go

```diff
@@ -0,0 +1,606 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (c) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more dtails.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.
+ *
+ */
+
+package builtin
+
+import (
+       "github.com/snapcore/snapd/interfaces"
+)
+
+var mirPermanentSlotAppArmor = []byte(`
+# Description: Allow operating as the Mir server. Reserved because this
+# gives privileged access to the system.
+# Usage: reserved
+
+
+capability dac_override,
+capability sys_tty_config,
+capability sys_admin,
+
+unix (receive, send) type=seqpacket addr=none,
```

Comment:
commenting out "unix (receive, send) type=seqpacket addr=none,"
results in the following upon trying to run the client app

Jun 16 20:56:11 localhost ubuntu-core-launcher[4427]: Loading module: 'libubuntu_application_api_desktop_mirclient.so.3.0.0'
Jun 16 20:56:12 localhost kernel: [ 3126.446100] audit: type=1400 audit(1466110572.256:20): apparmor="DENIED" operation="open" profile="snap.mir-client.client-start" name="/usr/share/applications/" pid=4429 comm="clocks" requested_mask="r" denied_mask="r" fsuid=0 ouid=0